gcalcli/gcal.py

```diff
@@ -79,10 +79,16 @@ def __init__(
         self.printer = printer
         self.options = options
         self.userless_mode = userless_mode
+        self.width = {}
 
         self.details = options.get('details', {})
-        # stored as detail, but provided as option: TODO: fix that
-        self.details['width'] = options.get('width', 80)
+
+        # Store overall calendar width and width for day table cells
+        self.width['cal'] = int(options.get('width', 80))
+        day_width = int(( self.width['cal'] - 8) / 7)
```

Comment:
FYI dividing by 7 probably doesn't handle `--noweekend` exactly right, but looks like that's a preexisting issue and minor.

Reply:
you're right with both: it's indeed not handled correctly and is a preexisting issue, but it got moved in this PR.
I've just opened a new PR #826 to fix it.